_i18n/fr/resources/user-guides/verification-windows-beginner.md

```diff
@@ -1,254 +1,253 @@
 
-![getkey right click](png/verify_binary_windows_beginner/verify-win_getkey-rightclick.png)
+![clic droit getkey](png/verify_binary_windows_beginner/verify-win_getkey-rightclick.png)
 
-Leave the default location, click `Save`.
+Laissez l'emplacement par défaut, supprimez l'extention `.txt` pour ne laisser comme nom de fichier que `fluffypony.asc` et cliquez sur `Enregistrer`.
 
 ![getkey save file](png/verify_binary_windows_beginner/verify-win_getkey-savefilename.png)
 
-### 2.2. Initialize Kleopatra
+### 2.2. Initialiser Kleopatra
 
-If this is your first time using Kleopatra you will have to create a key pair for yourself.
+S'il s'agit de votre première utilisation de Kleopatra vous devrez créer votre propre paire de clefs.
 
-Launch Kleopatra.
+Lancez Kleopatra.
 
-![kleo launch](png/verify_binary_windows_beginner/verify-win_kleopatra-launch.png)
+![lancement kleo](png/verify_binary_windows_beginner/verify-win_kleopatra-launch.png)
 
-#### 2.2.1. Import Signing Key
+#### 2.2.1. Importer une Clef de Signature
 
-Click `Import`.
+Cliquez sur `Importer...`.
 
-![kleo firstrun import](png/verify_binary_windows_beginner/verify-win_kleopatra-firstrun-importkey.png)
+![import démarrage kleo](png/verify_binary_windows_beginner/verify-win_kleopatra-firstrun-importkey.png)
 
-Enter the directory `Downloads`, select `fluffypony`, and click `Open`.
+Allez dans le répertoire  `téléchargements`, sélectionnez `fluffypony.asc` et cliquez sur `Ouvrir`.
 
-![kleo firstrun key location](png/verify_binary_windows_beginner/verify-win_kleopatra-firstrun-import-location.png)
+![emplacement clef démarrage kleo](png/verify_binary_windows_beginner/verify-win_kleopatra-firstrun-import-location.png)
 
-Start the process of certifying the key by clicking `Yes`.
+Démarrez le processus de certification de la clef en cliquant sur `Oui`.
 
-![kleo firstrun start process](png/verify_binary_windows_beginner/verify-win_kleopatra-firstrun-startverifyprocess.png)
+![certif process démarrage kleo](png/verify_binary_windows_beginner/verify-win_kleopatra-firstrun-startverifyprocess.png)
 
-#### 2.2.2. Create Key Pair
+#### 2.2.2. Créer la Paire de Clefs
 
-Start the process of key creation by clicking `Yes`.
+Démarrez le processus de création de clefs en cliquant sur `Oui`.
 
-![kleo firstrun start key create](png/verify_binary_windows_beginner/verify-win_kleopatra-firstrun-createkeysnow.png)
+![création clefs démarrage kleo](png/verify_binary_windows_beginner/verify-win_kleopatra-firstrun-createkeysnow.png)
 
-Fill in some details for `Name` and `Email`, click `Next`.
+Renseignez quelques information pour `Nom` et `Adresse électronique` et cliquez sur `suivant`.
 
-![kleo firstrun key details](png/verify_binary_windows_beginner/verify-win_kleopatra-firstrun-createkeydetails.png)
+![infos clefs démarrage kleo](png/verify_binary_windows_beginner/verify-win_kleopatra-firstrun-createkeydetails.png)
 
-Verify details, click `Create`.
+Vérifier les informations et cliquer sur `Créer`.
```

Comment:
"Vérifiez / cliquez" to be consistent with other steps.

Reply:
ok